% ****************************************************************************************************
% classicthesis-config-LyX.tex
% Use it in the preamble of your ClassicThesis.lyx
% ****************************************************************************************************
% If you like the classicthesis, we would appreciate a postcard.
% André's address can be found in the file ClassicThesis.pdf. A collection
% of the postcards received so far is available online at
% http://postcards.miede.de
% ****************************************************************************************************


% ****************************************************************************************************
% 1. Configure classicthesis for your needs here. At some point you will probably want to set 
% "drafting" to false in order to deactivate the time-stamp on the bottom of the pages
% See ClassicThesis.pdf for more information.
% ****************************************************************************************************
\PassOptionsToPackage{
	drafting=true,      % print version information on the bottom of the pages
	tocaligned=false,   % the left column of the toc will be aligned (no indentation)
	dottedtoc=false,    % page numbers in ToC flushed right
	parts=true,         % use part division
	eulerchapternumbers=true, % use AMS Euler for chapter font (otherwise Palatino)
	linedheaders=false,       % chaper headers will have line above and beneath
	floatperchapter=true,     % numbering per chapter for all floats (i.e., Figure 1.1)
	eulermath=false,    % use awesome Euler fonts for mathematical formulae (only with pdfLaTeX)
	beramono=true,      % toggle a nice monospaced font (w/ bold)
	palatino=false,      % toggle the standard roman font, see end of this file for more suggestions
	style=classicthesis % classicthesis, arsclassica
}{classicthesis}


% ****************************************************************************************************
% 2. Personal data and user ad-hoc commands
% ****************************************************************************************************
\newcommand{\myTitle}{TITLE\xspace}
\newcommand{\mySubtitle}{Doctoral dissertation\xspace}
\newcommand{\myDegree}{Master (MSc)\xspace}
\newcommand{\myName}{Józef Tran\xspace}
\newcommand{\myProf}{Artur Krężel\xspace}
% \newcommand{\myOtherProf}{Put name here\xspace}
% \newcommand{\mySupervisor}{Put name here\xspace}
\newcommand{\myFaculty}{Wydział Biotechnologii\xspace}
\newcommand{\myDepartment}{Department of Chemical Biology\xspace}
\newcommand{\myUni}{Uniwersytet Wrocławski\xspace}
\newcommand{\myLocation}{Wrocław\xspace}
\newcommand{\myTime}{February 2023\xspace}
\newcommand{\myVersion}{v1.0}

% There is a problem using ę with Mendeley, ę was expressed as \c{e}
\DeclareUnicodeCharacter{0229}{\k{e}}

% ****************************************************************************************************


% ****************************************************************************************************
% 3. Loading some handy packages
% ****************************************************************************************************
\usepackage{csquotes} % biblatex depends on these

\usepackage{scrhack} % fix warnings when using KOMA with listings package
\usepackage{xspace} % to get the spacing after macros right
\usepackage{mparhack} % get marginpar right
\PassOptionsToPackage{printonlyused,smaller}{acronym}
\usepackage{acronym} % nice macros for handling all acronyms in the thesis
%\renewcommand{\bflabel}[1]{{#1}\hfill} % fix the list of acronyms --> no longer working
%\renewcommand*{\acsfont}[1]{\textsc{#1}} 
%\renewcommand*{\aclabelfont}[1]{\acsfont{#1}}
\def\bflabel#1{{\acsfont{#1}\hfill}}
\def\aclabelfont#1{\acsfont{#1}}

% Package to use Python code in the thesis 
\usepackage[gobble=auto]{pythontex}
\def \thesispath{"C:\string\Users\string\makro\string\Desktop\string\PhD"}


\usepackage{newpxtext,newpxmath}
\useosf
\usepackage{textgreek}
\usepackage[version=4]{mhchem}

\hyphenation{mon-o-meric}

% \clubpenalty=9996
% \widowpenalty=9999
% \brokenpenalty=4991
% \predisplaypenalty=10000
% \postdisplaypenalty=1549
% \displaywidowpenalty=1602
% ****************************************************************************************************

% ****************************************************************************************************
% Glossary
% ****************************************************************************************************
% \usepackage[acronym]{glossaries}



\usepackage[automake,nolist,nopostdot,sort=use,acronym,]{glossaries}




\newglossarystyle{no_indentation_long}{% 
	\setglossarystyle{long}
	\renewenvironment{theglossary}%
	{\begin{longtable}[l]{@{}lp{\glsdescwidth}}}%
			{\end{longtable}}
}



%acronyms
%akronimy komentarze przykłady, to edytować - pozwala na robienie odnośników

\makeglossaries

%% modsuper glossary display style

% \newacronym{API}{API}{Application Programming Interface}
% \newglossaryentry{api}{type=\acronymtype, name={API}, description={Application Programming Interface}, first={Application Programming Interface (API)}}


\newglossaryentry{apig}{name={API},
	description={An Application Programming Interface (API) is a set
			of rules and protocols that a allows two or more software components
			to communicate with each other.}}

\newglossaryentry{api}{type=\acronymtype, name={API},
	description={Application
			Programming Interface, \glsseeformat[Glossary:]{apig}{}},
	first={Application Programming Interface (API)\glsadd{apig}}}




\newglossaryentry{ppi}{type=\acronymtype, name={PPI},
	description={Protein-Protein Interaction},
	first={Protein-Protein Interaction (PPI)},
	plural={PPIs},
	descriptionplural={Protein-Protein Interactions},
	firstplural={Protein-Protein Interactions (\glsentryplural{ppi})}}



\newglossaryentry{proteome}{name={proteome},
	description={A proteome is the complete set of poroteins
			that can or are expressed by an organism.},}

\newglossaryentry{htsg}{name={HTS},
	description={High-throughput screening is a application
			of robotics followed by data processing to test large
			number of samples.}}

\newglossaryentry{hts}{type=\acronymtype, name={HTS},
	description={High-Throughput Screening, \glsseeformat[Glossary:]{htsg}{}},
	first={High-Throughput Screening (HTS)\glsadd{htsg}}}

\newglossaryentry{interface}{name={interface},
	description={A shared surface between two or more components.
			Interface in \gls{ppi} is made of residues of a protein that
			contact with residues from another interacting partner. Sometimes
			interfaces may include additional interaction partners, e.g. protein cofactors
			and ligands.}}

\newglossaryentry{zppig}{name={ZPPI},
	description={Zn(II)-involved \glsdesc{ppi} is a protein-protein interaction
			in which a vital role is played by Zn(II)},}

\newglossaryentry{zppi}{type=\acronymtype, name={ZPPI},
	description={Zn(II)-involved \glsdesc{ppi}, \glsseeformat[Glossary:]{zppig}{}},
	first={Zn(II)-involved \glsdesc{ppi} (ZPPI)\glsadd{zppig}},
	plural={ZPPIs},
	descriptionplural={Zn(II)-involved \glsdescplural{ppi}},
  firstplural={Zn(II)-involved \glsdescplural{ppi} (\glsentryplural{zppi})}}

\newglossaryentry{cry}{type=\acronymtype, name={CRY1},
	description={Cryptochrome 1},
	first={Cryptochrome 1 (CRY1)},
}

\newglossaryentry{per}{type=\acronymtype, name={PER2},
	description={Period 2},
	first={Period 2 (PER2)}
}


\newglossaryentry{freezn}{name={free Zn(II)},
	description={Available, loosely bound Zn(II) ion.},}

\newglossaryentry{rcsb}{type=\acronymtype, name={RCSB PDB},
description={Research Collaboratory for Structural Bioinformatics Protein Data Bank},
first={Research Collaboratory for Structural Bioinformatics \gls{pdb} (RCSB \gls{pdb})}
}

\newglossaryentry{pdb}{type=\acronymtype, name={PDB},
description={Protein Data Bank or Protein Data Bank file type},
first={Protein Data Bank (PDB)}
}

\newglossaryentry{python}{name={Python},
	description={Python is a high-level, object-oriented programming
	language, often used in scientific community.},}

\newglossaryentry{bassembly}{name={biological assembly},
description={The biological assembly is the arrangement of macromolecules
in space that is believed or proved to be the functional form of the molecule.
The biological assembly may be a part of the \gls{asunit}, as well as \gls{asunit} may contain one
or more copies of the biological assembly.},
plural = {biological assemblies}}

\newglossaryentry{asunit}{name={assymetric unit},
description={The asymmetric unit is the basic unit ot a crystal structure. After application
symmetry operations to the asymmetric unit it is possible to generate the unit cell, which duplicated and translated can produce the whole crystal.},}

\newglossaryentry{code}{name={source code},
description={Source code is the set of instuctions written using programming language (e.g. \gls{python}.)},}

\newglossaryentry{fold}{name={protein fold},
description={Definition of a protein fold is arbitrary and depending on a context.
Generally protein fold is usually understood as a comon shape for proteins.}}



\newglossaryentry{radg}{name={Rad50},
description={Components of the \gls{mrn} complex.
The complex plays a key role in \gls{dsb} repair.}}

\newglossaryentry{rad}{type=\acronymtype,
	name={Rad50},
	description={DNA rapair protein Rad50, \glsseeformat[Glossary:]{radg}{}},
	first={DNA rapair protein Rad50 (Rad50)\glsadd{radg}},
}


\newglossaryentry{mrng}{name={MRN(X)},
description={The \gls{mrn} complex plays a key in \gls{dsb} repair.}}

\newglossaryentry{mrn}{type=\acronymtype,
name={MRN(X)},
description={\gls{mre}-\gls{rad}-\gls{nbs}(\gls{xrs}) \glsseeformat[Glossary:]{mrng}{}},
first={\gls{mre}-\gls{rad}-\gls{nbs}(\gls{xrs}) complex\glsadd{mrng}},
}



\newglossaryentry{mreg}{name={Mre11},
description={Components of the \gls{mrn} complex.
\Gls{mre} provides both, single-strand endonuclease
and double-strand-specific 3'-5' exonuclease activity to the \gls{mrn}.}}

\newglossaryentry{mre}{type=\acronymtype,
name={Mre11},
description={Double-strand break repair protein Mre11, \glsseeformat[Glossary:]{mreg}{}},
first={Double-strand break repair protein Mre11 (Mre11)\glsadd{mreg}},
}


\newglossaryentry{nbsg}{name={Nbs1},
description={Components of the \gls{mrn} complex.
\Gls{nbs} modulates the DNA damage repair by recruiting various kinases..}}

\newglossaryentry{nbs}{type=\acronymtype,
name={Mre11},
description={Nibrin, \glsseeformat[Glossary:]{nbsg}{}},
first={Nibrin (Nbs1)\glsadd{nbsg}},
}



\newglossaryentry{xrsg}{name={Xrs2},
description={Components of the \gls{mrn} complex.
\Gls{xrs} and \gls{nbs} share little sequence identity.
\gls{xrs} probably regulates the 5'-3' exonuclease activity of the \gls{mrn} complex.}}

\newglossaryentry{xrs}{type=\acronymtype,
name={Xrs2},
description={DNA repair protein XRS2, \glsseeformat[Glossary:]{xrsg}{}},
first={DNA repair protein (Xrs2)\glsadd{xrsg}}}

\newglossaryentry{dsb}{type=\acronymtype,
name={DSB},
description={Double-Strand Break},
first={Double-Strand Break (DSB)},
plural={DSBs},
descriptionplural={Double-Strand Breaks},
firstplural={Double-Strand Breaks (\glsentryplural{dsb})}
}

\newglossaryentry{hs}{type=\acronymtype,
name={\textit{H. sapiens}},
description={\textit{Homo sapiens}},
first={\textit{Homo sapiens} (\textit{H. sapiens})}}

\newglossaryentry{pf}{type=\acronymtype,
name={\textit{P. furiosus}},
description={\textit{Pyrococcus furiosus}},
first={\textit{Pyrococcus furiosus} (\textit{P. furiosus})}}


\newglossaryentry{cd}{type=\acronymtype,
name={CD},
description={Circular Dichroism},
first={Circular Dichroism (CD)}}

\newglossaryentry{saxs}{type=\acronymtype,
name={SAXS},
description={Small-angle X-ray scattering},
first={Small-angle X-ray scattering (SAXS)}}

\newglossaryentry{xas}{type=\acronymtype,
name={XAS},
description={X-ray absorption spectroscopy},
first={X-ray absorption spectroscopy (XAS)}}

\newglossaryentry{hsabg}{name={HSAB},
	description={Hard and Soft Acid and Bases principle describes what acids and bases prefer to interact.
	Soft metal ions (Lewis acids) will preferentially interact with soft Lewis bases, e.g. Hg(II) will
	preferentially react with thiolates.},}

\newglossaryentry{hsab}{type=\acronymtype,
	name={HSAB},
	description={Hard and Soft Acid and Bases , \glsseeformat[Glossary:]{hsabg}{}},
	first={Hard and Soft Acid and Bases (HSAB)\glsadd{hsabg}},
	}

\newglossaryentry{merr}{type=\acronymtype,
name={MerR},
description={Mercuric resistance operon regulatory protein},
first={Mercuric resistance operon regulatory protein (MerR)},
}

\newglossaryentry{atox}{type=\acronymtype,
name={ATOX1},
description={Copper transport protein ATOX1},
first={Copper transport protein ATOX1 (ATOX1)},
}

\newglossaryentry{uvvis}{type=\acronymtype,
name={UV-Vis},
description={Ultraviolet-Visible},
first={Ultraviolet-Visible (UV-Vis)}
}

\newglossaryentry{pac}{type=\acronymtype,
name={PAC},
description={Perturbed \textgamma-\textgamma\ Angular Correlation},
first={Perturbed \textgamma-\textgamma\ Angular Correlation (PAC)}
}

\newglossaryentry{nmr}{type=\acronymtype,
name={NMR},
description={Nuclear Magnetic Resonance},
first={Nuclear Magnetic Resonance (NMR)}
}


\newglossaryentry{DTT}{type=\acronymtype,
name={DTT},
description={dithiothreitol},
first={dithiothreitol (DTT)}
}

\newglossaryentry{itc}{type=\acronymtype,
name={ITC},
description={Isothermal Titration Calorimetry},
first={Isothermal Titration Calorimetry (ITC)}
}


\newglossaryentry{dG}{type=\acronymtype,
name={\textDelta \textit{G}\textdegree},
description={Gibbs free energy change at standard state},
first={Gibbs free energy change at standard state (\textDelta \textit{G}\textdegree)}
}

\newglossaryentry{dH}{type=\acronymtype,
name={\textDelta \textit{H}\textdegree},
description={Enthalpy change at standard state},
first={Enthalpy at standard state (\textDelta \textit{H}\textdegree)}
}

\newglossaryentry{dS}{type=\acronymtype,
name={\textDelta \textit{S}\textdegree},
description={Entropy change at standard state},
first={Entropy change at standard state (\textDelta \textit{S}\textdegree)}
}

\newglossaryentry{T}{type=\acronymtype,
name={\textit{T}},
description={absolute temperature},
first={absolute temperature (\textit{T})}
}

\newglossaryentry{K12}{type=\acronymtype,
name={\textit{K}\textsubscript{12}},
description={cumulative formation constant of the \glslink{Me}{Me}\glslink{L}{L}\textsubscript{2} complex},
first={cumulative formation constant of the \glslink{Me}{Me}\glslink{L}{L}\textsubscript{2} complex (\textit{K}\textsubscript{12})}
}

\newglossaryentry{Me}{type=\acronymtype,
name={Me},
description={metal},
first={metal (Me)}
}

\newglossaryentry{Lg}{name={ligand},
	description={In chemistry ligand is a ion or molecule that act as a Lewis base
	(see \gls{hsab}) for the molecule or ion (a Lewis acid). Thus interacting
	with the Lewis acid by a coordination bond.
	In biochemistry  ligand is understood as a molecule that binds to another
	(macro)molecule.}}

\newglossaryentry{L}{type=\acronymtype, name={L},
	description={ligand, \glsseeformat[Glossary:]{Lg}{}},
	first={ligand (L) \glsadd{Lg}},
	descriptionplural ={ligands}}

\newglossaryentry{R}{type=\acronymtype,
name={R},
description={molar gas constant},
first={molar gas constant (R)}
}

\newglossaryentry{agnp}{type=\acronymtype, name={AgNP},
	description={silver nanoparticle},
	first={silver nanoparticle (AgNP)},
	descriptionplural={silver nanoparticles},
	firstplural = {silver nanoparticles (AgNPs)}
	}

\newglossaryentry{hmw}{type=\acronymtype, name={HMW},
description={high molecular weight},
first={high molecular weight (HMW)},
}


\newglossaryentry{ptmg}{name={PTM},
	description={Post-translational modification is the covalent (usually enzymatic)
	modification of protein that is subsequent to the protein biosynthesis.}}

\newglossaryentry{ptm}{type=\acronymtype, name={PTM},
	description={post-translational modification, \glsseeformat[Glossary:]{ptmg}{}},
	first={post-translational modification (PTM) \glsadd{ptmg}},
	plural={PTMs},
	descriptionplural={post-translational modifications},
	firstplural={post-translational modifications (\glsentryplural{ptm})}}

\newglossaryentry{coenzyme}{name={coenzyme},
description={Coenzyme is a non-protein organic \gls{cofactor}.}} %https://www.jove.com/science-education/10975/cofactors-and-coenzymes

\newglossaryentry{cofactor}{name={cofactor},
description={Cofactor is a non-protein chemical compound required for a protein to fulfill its functions.}} 

\newglossaryentry{pgroup}{name={prosthetic group},
description={Prosthetic group is a non-protein compound required for a protein to fulfill its functions,
however, contrary to the \gls{cofactor} prosthetic group is covalently conjugated with a protein.}} 


\newglossaryentry{mprotein}{name={metalloprotein},
description={metalloprotein is a protein that contain at least one metal ion as a \gls{cofactor} or as a part of \gls{pgroup}.}} 


\newglossaryentry{atp}{type=\acronymtype, name={ATP},
	description={adenosine triphosphate},
	first={adenosine triphosphate (ATP)},
	}

\newglossaryentry{sod}{type=\acronymtype, name={SOD},
description={superoxide dismutase},
descriptionplural={superoxide dismutases},
first={superoxide dismutase (SOD)},
firstplural={superoxide dismutases (SODs)},
}

\newglossaryentry{dna}{type=\acronymtype, name={DNA},
	description={deoxyribonucleic acid},
	first={deoxyribonucleic acid (DNA)},
	}

\newglossaryentry{ca2}{type=\acronymtype, name={CAII},
description={carbonic anhydrase II},
first={carbonic anhydrase II (CAII)},
}

\newglossaryentry{cn}{type=\acronymtype, name={CN},
description={coordination number},
first={coordination number (CN)},
firstplural={coordination numbers (CNs)},
plural={CNs},
}

%%%% shortcuts
\newcommand{\gshort}[1]{\glslink{#1}{#1}}
% affinity

% ****************************************************************************************************
% 4. Setup floats: tables, (sub)figures, and captions
% ****************************************************************************************************
\usepackage{tabularx} % better tables
\setlength{\extrarowheight}{3pt} % increase table row height
\newcommand{\tableheadline}[1]{\multicolumn{1}{l}{\spacedlowsmallcaps{#1}}}
\usepackage{subfig}
% ****************************************************************************************************


% ****************************************************************************************************
% 5. Setup code listings
% ****************************************************************************************************
\usepackage{listings}
%\lstset{emph={trueIndex,root},emphstyle=\color{BlueViolet}}%\underbar} % for special keywords
\lstset{language=[LaTeX]Tex,%C++,
	keywordstyle=\color{RoyalBlue},%\bfseries,
	basicstyle=\small\ttfamily,
	%identifierstyle=\color{NavyBlue},
	commentstyle=\color{Green}\ttfamily,
	stringstyle=\rmfamily,
	numbers=none,%left,%
	numberstyle=\scriptsize,%\tiny
	stepnumber=5,
	numbersep=8pt,
	showstringspaces=false,
	breaklines=true,
	frameround=ftff,
	frame=single,
	belowcaptionskip=.75\baselineskip
	%frame=L
}
% ****************************************************************************************************


% ****************************************************************************************************
% 6. Last calls before the bar closes
% ****************************************************************************************************
% ********************************************************************
% Her Majesty herself
% ********************************************************************
\usepackage{classicthesis}


% ********************************************************************
% Fine-tune hyperreferences (hyperref should be called last)
% ********************************************************************
\hypersetup{%
	%draft, % hyperref's draft mode, for printing see below
	colorlinks=true, linktocpage=true, pdfstartpage=3, pdfstartview=FitV,%
	% uncomment the following line if you want to have black links (e.g., for printing)
	%colorlinks=false, linktocpage=false, pdfstartpage=3, pdfstartview=FitV, pdfborder={0 0 0},%
	breaklinks=true, pdfpagemode=UseNone, pageanchor=true, pdfpagemode=UseNone,%
	plainpages=false, bookmarksnumbered, bookmarksopen=true, bookmarksopenlevel=1,%
	hypertexnames=true, pdfhighlight=/O,%nesting=true,%frenchlinks,%
	urlcolor=CTurl, linkcolor=CTlink, citecolor=CTcitation,%
	%urlcolor=Black, linkcolor=Black, citecolor=Black,%
	pdftitle={\myTitle},%
	pdfauthor={\textcopyright\ \myName, \myUni, \myFaculty},%
	pdfsubject={},%
	pdfkeywords={},%
	pdfcreator={},%
	pdfproducer={LaTeX with classicthesis style}%
}


% ********************************************************************
% Setup autoreferences (hyperref and babel)
% ********************************************************************
% There are some issues regarding autorefnames
% http://www.ureader.de/msg/136221647.aspx
% http://www.tex.ac.uk/cgi-bin/texfaq2html?label=latexwords
% you have to redefine the macros for the
% language you use, e.g., american, ngerman
% (as chosen when loading babel/AtBeginDocument)
% ********************************************************************
\makeatletter
\@ifpackageloaded{babel}%
{%
	\addto\extrasamerican{%
		\renewcommand*{\figureautorefname}{Figure}%
		\renewcommand*{\tableautorefname}{Table}%
		\renewcommand*{\partautorefname}{Part}%
		\renewcommand*{\chapterautorefname}{Chapter}%
		\renewcommand*{\sectionautorefname}{Section}%
		\renewcommand*{\subsectionautorefname}{Section}%
		\renewcommand*{\subsubsectionautorefname}{Section}%
	}%
	\addto\extrasngerman{%
		\renewcommand*{\paragraphautorefname}{Absatz}%
		\renewcommand*{\subparagraphautorefname}{Unterabsatz}%
		\renewcommand*{\footnoteautorefname}{Fu\"snote}%
		\renewcommand*{\FancyVerbLineautorefname}{Zeile}%
		\renewcommand*{\theoremautorefname}{Theorem}%
		\renewcommand*{\appendixautorefname}{Anhang}%
		\renewcommand*{\equationautorefname}{Gleichung}%
		\renewcommand*{\itemautorefname}{Punkt}%
	}%
	% Fix to getting autorefs for subfigures right (thanks to Belinda Vogt for changing the definition)
	\providecommand{\subfigureautorefname}{\figureautorefname}%
}{\relax}
\makeatother

% ****************************************************************************************************
% 7. Further adjustments (experimental)
% ****************************************************************************************************
% ********************************************************************
% Changing the text area
% ********************************************************************
%\areaset[current]{312pt}{761pt} % 686 (factor 2.2) + 33 head + 42 head \the\footskip
%\setlength{\marginparwidth}{7em}%
%\setlength{\marginparsep}{2em}%

% ********************************************************************
% Using different fonts
% This is for pdflatex; xelatex and lualatex have their own way
% ********************************************************************
%\usepackage[oldstylenums]{kpfonts} % oldstyle notextcomp
%\usepackage[osf]{libertine}
%\usepackage[light,condensed,math]{iwona}
%\renewcommand{\sfdefault}{iwona}
%\usepackage{lmodern} % <-- no osf support :-(
%\usepackage{cfr-lm} %
%\usepackage[urw-garamond]{mathdesign} <-- no osf support :-(
%\usepackage[default,osfigures]{opensans} % scale=0.95
%\usepackage[sfdefault]{FiraSans}
%\usepackage[opticals,mathlf]{MinionPro} % onlytext
% ********************************************************************
% \usepackage[largesc,osf]{newpxtext}
\linespread{1.05} % a bit more for Palatino
% Used to fix these:
% https://bitbucket.org/amiede/classicthesis/issues/139/italics-in-pallatino-capitals-chapter
% https://bitbucket.org/amiede/classicthesis/issues/45/problema-testatine-su-classicthesis-style
% ********************************************************************
%\linespread{1.05} % a bit more for Palatino
% ****************************************************************************************************
